microsetta_public_api/models/_exceptions.py

```diff
@@ -0,0 +1,6 @@
+class UnknownID(ValueError):
```

Comment:
Flask can automatically convert registered exception types into specific error messages sent down over the API.  We probably could use more thought on this in the private api.  Are you going to have UnknownID exceptions automatically turn into 404s?

Reply:
Are you saying by doing something like this page in the docs? https://flask.palletsprojects.com/en/1.1.x/patterns/apierrors/